opm/utility/ECLGraph.cpp

```diff
@@ -1855,6 +1940,69 @@ flux(const PhaseIndex phase) const
     return v;
 }
 
+namespace Opm {
+
+    template <typename T>
+    std::vector<T>
+    ECLGraph::Impl::rawLinearisedCellData(const std::string& vector) const
+    {
+        auto x = std::vector<T>{};  x.reserve(this->numCells());
+
+        for (const auto& G : this->grid_) {
```

Comment:
The capital letter G is a MRST-ism, we should use lower-case letters for variable names.

Not requiring a change in this PR, as this already occurs in the code base and if changed it should be done in a consistent manner.